for some reason the or doesn't work and it only uses the former and fails
--
diff --git a/_build/cl-gobject-introspection/src/init.lisp b/_build/cl-gobject-introspection/src/init.lisp
index a335f3c..e3507c0 100644
--- a/_build/cl-gobject-introspection/src/init.lisp
+++ b/_build/cl-gobject-introspection/src/init.lisp
@@ -10,7 +10,7 @@
     (t "libgobject-2.0"))
   (cffi:define-foreign-library girepository
     (:darwin "libgirepository-1.0.dylib")
-    (:unix (:or "libgirepository-1.0.so" "libgirepository-1.0.so.1"))
+    (:unix (:or "libgirepository-1.0.so.1"))
     (:windows (:or "libgirepository-1.0.dll" "libgirepository-1.0.0.dll"
                    "libgirepository-1.0-1.dll"))
     (t "libgirepository-1.0")))
